zerver/views/registration.py

```diff
@@ -465,10 +468,28 @@ def accounts_register(request: HttpRequest) -> HttpResponse:
             "MAX_NAME_LENGTH": str(UserProfile.MAX_NAME_LENGTH),
             "MAX_PASSWORD_LENGTH": str(form.MAX_PASSWORD_LENGTH),
             "MAX_REALM_SUBDOMAIN_LENGTH": str(Realm.MAX_REALM_SUBDOMAIN_LENGTH),
+            "EMAIL_ADDRESS_VISIBILITY_EVERYONE": email_address_visibility,
+            "EMAIL_ADDRESS_VISIBILITY_ADMINS": email_address_visibility,
+            "EMAIL_ADDRESS_VISIBILITY_NOBODY": email_address_visibility,
+            "EMAIL_ADDRESS_VISIBILITY": email_address_visibility,
         },
     )
 
 
+def get_email_address_visibility(realm: Optional[Realm]) -> str:
+    # print(str(realm.email_address_visibility))
+    if realm is None:
+        return "All users will be able to see this email address."
+    if realm.email_address_visibility == Realm.EMAIL_ADDRESS_VISIBILITY_EVERYONE:
+        return "All users will be able to see this email address."
+    elif realm.email_address_visibility == Realm.EMAIL_ADDRESS_VISIBILITY_ADMINS:
+        return "Only organization administrators will be able to see this email address."
+    elif realm.email_address_visibility == Realm.EMAIL_ADDRESS_VISIBILITY_NOBODY:
+        return "Users will not be able to see this email address."
+    else:
+        return "All users will be able to see this email address."
```

Comment:
I think instead of returning the same string at multiple places, you can store all the strings in some variables and return those variables instead.

Doing this, if in case in future we need to change something in any of the string, we would only have to change it at a single place and not at multiple places.

Reply:
Updated!